### PR TITLE
[FIX JENKINS-33804] adding style to banner

### PR DESCRIPTION
**Pull request by gusreiber**

### CODE REVIEW

**Comment by gusreiber** [comment]
@reviewbybees
...some art goop.

**Comment by kzantow** [comment]
screenshot? (i meant to get to this... whoops)

**Comment by gusreiber** [comment]
screen shot 2016-03-29 at 7 52 29 pm
....drrrr..... oooops. yes.

**Comment by kzantow** [comment]
Soooooooooo pretty :)

:bee:

File: core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly [hunk 1]
```diff
@@ -3,25 +3,52 @@
...
         font-size: 20px;
         height: 60px;
-
+        vertical-align:middle;
         display: flex;
         justify-content: center;
         align-items: center;
       }
-
+      #upgrade-notice > .post{
```
Comment (recena) [inline core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly]: Where is being used `.post`?

Comment (recena) [inline core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly]: Generated by `<f:link >`?

Comment (gusreiber) [inline core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly]: @recena  yes, that is correct

File: core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly [hunk 2]
```diff
@@ -3,25 +3,52 @@
...
       #upgrade-notice {
-        font-weight: bold;
         text-align: center;
         font-size: 20px;
         height: 60px;
-
+        vertical-align:middle;
         display: flex;
         justify-content: center;
         align-items: center;
```
Comment (recena) [inline core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly]: I know that it was introduced before but... really needed for this banner?

Comment (gusreiber) [inline core/src/main/resources/jenkins/install/UpgradeWizard/footer.jelly]: It is, yes. The banner text is chopped into in-line blocks so as to get the button in there. We could do math to align the baseline of the text in the button to match the baseline of the text. We could use line-height, but vertical-align middle also does the job. Not sure what the harm is we might be concerned about.

**Pull request merged**